Tests/DatadogTests/Datadog/RUM/WebView/WKUserContentController+DatadogTests.swift

```diff
@@ -96,6 +102,31 @@ class WKUserContentController_DatadogTests: XCTestCase {
         XCTAssertEqual(recordedLogMessages, Array(repeating: "`trackDatadogEvents(in:)` was called more than once for the same WebView. Second call will be ignored. Make sure you call it only once.", count: multipleTimes - 1))
     }
 
+    func testWhenStoppingTracking_itKeepsNonDatadogComponents() throws {
+        let controller = DDUserContentController()
+
+        controller.trackDatadogEvents(in: [])
+
+        let componentCount = 10
```

Comment:
why make it 10? I guess adding a single or a couple of other scripts will cover this change